### PR TITLE
CA-27 Compatibility with networking module

### PR DESCRIPTION
**Pull request by franco-caylent**
- Separated subnets for Data, Application and Compute.
- Moved RDS and ES to the Data subnet
- Moved TamrVM to Application subnet
- Moved EMR to Compute subnet
- Fixed Tamr-VM defaulting to us-east-1

### CODE REVIEW

File: examples/minimal/tamr-vm.tf [hunk 1]
```diff
@@ -2,12 +2,12 @@ module "tamr-vm" {
...
 
   ami                = var.ami_id
-  instance_type      = "m4.2xlarge"
+  instance_type      = "r5.2xlarge"
   key_name           = module.emr_key_pair.key_pair_key_name
   subnet_id          = var.ec2_subnet_id
   vpc_id             = var.vpc_id
   security_group_ids = module.aws-sg-vm.security_group_ids
-
+  availability_zone = data.aws_subnet.application_subnet.availability_zone
```
Comment (souza-dan) [inline examples/minimal/tamr-vm.tf:10]: Formatting seems off here, you may need to run `terraform fmt`

**Review by souza-dan — APPROVED** [review]
Content is great! Just 1 nit about formatting.

File: examples/minimal/tamr-vm.tf (hunk 1 above)
Comment (franco-caylent) [reply examples/minimal/tamr-vm.tf]: Fixed [here](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/20f40c013e2fa8cc62d9a987df2deb829e6cd98d). Also [added a variable](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/1f9516c4214df62726578e886e889dc16526ce5b) that was missing

File: examples/minimal/rendered-config.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+"APPS_DMS_DEFAULT_CLOUD_PROVIDER": "s3"
```
Comment (souza-dan) [inline examples/minimal/rendered-config.yml:1]: No need to check in the rendered config. Seems worthwhile to add to the .gitignore file.

Comment (franco-caylent) [reply examples/minimal/rendered-config.yml]: [Deleted](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/16e42000a8edb4c853644446e1be70d1e6a4b02a) and [added to gitignore](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/4cf61e31dce5d04af4641b75b62f0eba87401a65)

**Pull request closed without merging**